src/examples/rest/src/backend/schema/auth/api-key/entity.ts

```diff
@@ -0,0 +1,36 @@
+import {
+	AccessControlList,
+	ApiKeyStorage,
+	ApplyAccessControlList,
+	AuthorizationContext,
+} from '@exogee/graphweaver-auth';
+import { ID, ObjectType, Field, SummaryField, ReadOnly, GraphQLEntity } from '@exogee/graphweaver';
+
+import { ApiKey as OrmApiKey } from '../../../entities';
+
+const acl: AccessControlList<ApiKey, AuthorizationContext> = {
+	DARK_SIDE: {
+		// Dark side user role can perform operations on any api keys
+		all: true,
+	},
+};
+
+@ReadOnly({ adminUI: false, backend: true })
```

Comment:
Note: when I was working with Dylan yesterday, the decorator with the  `{ adminUI: false, backend: true }` options that was in his version of GW was `@ReadOnlyProperty` his no longer had `@ReadOnly`.

Reply:
I guess that was an older version?